webpack.config.js

```diff
@@ -19,9 +18,14 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
         publicDir = composerConfig.extra['public-dir'];
     }
 
-    const entries = glob.sync(
-        path.resolve(__dirname, 'src/Sulu/Bundle/*/Resources/js/index.js') // eslint-disable-line no-undef
-    );
+    const entries = [];
+
+    entries.unshift('sulu-admin-bundle');
+    entries.unshift('sulu-page-bundle');
+    entries.unshift('sulu-media-bundle');
+    entries.unshift('sulu-preview-bundle');
+    entries.unshift('sulu-security-bundle');
```

Comment:
I am still not sure if I would remove `glob` here... Actually I even thought about expanding the `glob`, so that even 3rd party bundles would not have to touch the webpack config, making it easier for developers to start...

Reply:
The glob is the evil about the problems moving to assets/admin and have to define all dependencies also in the project because NPM think this not third party libraries and don't find its dependencies correctly.

Reply:
Okay, that makes sense... However, you would still have to add `sulu-contact-bundle` now, since it was recently added :-)

Reply:
@danrot added